src/Altinn.App.Api/Controllers/InstancesController.cs

```diff
@@ -844,6 +844,8 @@ private async Task CopyDataFromSourceInstance(ApplicationMetadata application, I
 
                     await _instantiationProcessor.DataCreation(targetInstance, data, null);
 
+                    ObjectUtils.InitializeAltinnRowId(data);
```

Comment:
Candidate method for moving to a future DataService implementation instead of Utils class, but that should also include refactoring  this controller to not use the IDataClient directly but rather use the DataService and perhaps refactor even more logic out from the controller. But not critical now I think.

Reply:
Yes, out of scope now, but would be great if we had test coverage to ensure that the future rewrite doesn't break the behavior.